jenkins/build.jpl

```diff
@@ -115,9 +118,22 @@ node("builder") {
         }
     }
 
-    stage("Build") {
-        timeout(time: 180, unit: 'MINUTES') {
-            buildConfig(params.DEFCONFIG, kdir, kci_build)
+    stage("Delay") {
```

Comment:
As discussed on IRC, I think this should be split into a separate commit and not put in production unless we really can't fix the builders.  I guess it could stay on a wip branch to experiment and investigate disk i/o issues on staging.

Reply:
I'll just pull the delay support out